### ISSUE TITLE
Show main waypoint's restriction access field on outing

### ISSUE THREAD

**Issue by cbeauchesne** [c2corg/c2c_ui#404]
> Il faudrait que ces restrictions d'accès soient aussi affichées sur les itinéraires dans le cas des couennes que l'on ait qu'un champ à gérer : celui du WP.

Need to add this field in locales on associated waypoints

**Comment by cbeauchesne** [comment]
Here is the expected data structure from API : 

```
route
   associations
       waypoints[n]
            document_id: <waypoint's ID
            locales[n]
                title: <waypoint's title>
                access_period: <markdown>
            cooked
                title: <waypoint's title>
                access_period: <markdown>
```

**Comment by brunobesson** [comment]
API is updated. Ready to dev.

**Comment by brunobesson** [comment]
Alas.... not. We do not generate cooked value 🙁

**Comment by cbeauchesne** [comment]
We use a naïve markdown interpreter for the summary: 

* We may use it waiting for the API modification ? 
* when we'll do the API modification, it'd be a good idea to also cook the summary

**Comment by brunobesson** [comment]
> We may use it waiting for the API modification ?

You're right, the case where this field is more than a paragraph must be an exception. Maybe with the exception of links?
